mempool/mempool.go

```diff
@@ -112,6 +112,7 @@ type PostCheckFunc func(types.Tx, *abci.ResponseCheckTx) error
 // to the expected maxBytes.
 func PreCheckMaxBytes(maxBytes int64) PreCheckFunc {
 	return func(tx types.Tx) error {
+		// we subtract 4 here to account for the extra fields added to the data message
```

Comment:
No other change happens here (w.r.t. `v0.34.20`), so what does this comment refer to?

Reply:
It seems residual so I created https://github.com/celestiaorg/celestia-core/pull/907